src/build/mdn-comments.ts

```diff
@@ -110,3 +111,25 @@ export async function generateDescription(): Promise<Record<string, string>> {
 
   return {};
 }
+export function extractSummaryFromFile(url: string): string {
+  const relativePath = url
+    .replace("https://developer.mozilla.org/docs/", "")
+    .split("#")[0]
+    .toLowerCase();
```

Comment:
We should compute the path rather than using the URL this way. Basically for `Foo.bar` the path should be `/api/foo/bar/index.md`.

And why do we need this when we already have `generateDescriptions` above that collects index files?